### PR TITLE
changelog and more info pages update

### PR DESCRIPTION
**Pull request by ma-zah**
pls review.

### CODE REVIEW

File: changelog.mdx [hunk 1]
```diff
@@ -4,6 +4,144 @@ description: "All the nice stuff you asked for"
...
+    src="images/setup/setup-11-suggest-more.png"
+    alt="have AI suggest more test cases"
+  />
+</Frame>
+
+- `Open status for new test cases`: In order to mark test cases coming out of **suggest more** as new (until you've opened them for the first time), we introduced a new column **open status** to properly persist the state.
+
+<Frame caption="Open status on new suggested tests, 08/2024">
+  <img
+    src="images/changelog/open-status-proposals.png"
```
Comment (sker65) [inline changelog.mdx:55]: in the open status proposal the test case list view is inconsistent to the one above: two different suggest more buttons. consider to change the  open-status-proposals.png to show current state.

File: changelog.mdx [hunk 2]
```diff
@@ -4,6 +4,144 @@ description: "All the nice stuff you asked for"
...
+- `AI suggesting more tests`: Similar to the AI discovery at set-up, we expanded the AI discovery feature to “spawn” individual test cases.
+
+<Frame caption="Have the AI agent suggest more test cases, 08/2024">
+  <img
+    src="images/setup/setup-11-suggest-more.png"
+    alt="have AI suggest more test cases"
+  />
+</Frame>
+
+- `Open status for new test cases`: In order to mark test cases coming out of **suggest more** as new (until you've opened them for the first time), we introduced a new column **open status** to properly persist the state.
```
Comment (sker65) [inline changelog.mdx:51]: "... to properly persist state" !?!? why not just we introduce a new column.

File: changelog.mdx [hunk 3]
```diff
@@ -4,6 +4,144 @@ description: "All the nice stuff you asked for"
...
+  <img
+    src="images/editing/save-run-only.png"
+    alt="Run only to validate the test before saving it"
+    width="300px"
+  />
+</Frame>
+
+## 2024-08-06
+
+- `AI suggesting more tests`: Similar to the AI discovery at set-up, we expanded the AI discovery feature to “spawn” individual test cases.
```
Comment (sker65) [inline changelog.mdx:42]: maybe just: to spawn **more** test cases?

File: changelog.mdx [hunk 4]
```diff
@@ -4,6 +4,144 @@ description: "All the nice stuff you asked for"
...
+  />
+</Frame>
+
+## 2024-07-04
+
+- `Mobile + tablet use`: Test case editing works better on smaller screens now.
+
+## 2024-07-01
+
+- `Agent benchmark automation`: Every major change in the codebase impacts the AI agent performance. We get an automated ping whether the agent does what it icons supposed to, now.
```
Comment (sker65) [inline changelog.mdx:127]: whether the agent does what it _icons_ supposed to, now.   ?!?

File: more-info/octo-dictionary.mdx [hunk 5]
```diff
@@ -80,3 +80,14 @@ to book a hotel on a hotel booking page.
 <Frame caption="example of a user flow, 03/2024">
   <img src="/images/user-flow.png" alt="example of a user flow" />
 </Frame>
+
+### Virtual locator picker
```
Comment (sker65) [inline more-info/octo-dictionary.mdx:84]: Virtual or Visual?

File: more-info/under-the-hood.mdx [hunk 6]
```diff
@@ -4,31 +4,41 @@ description: "If you are interested in how we make the tests work, this is your
...
-The corresponding [Playwright](https://playwright.dev/) code which is exercising the UI is generated on the fly just before test execution. The interaction chain can be examined in the test detail view and the Playwright trace viewer.
 
-## Auto-maintenance
+Our AI agents mimic human users (i.e., clicks input fields, signs up for newsletter) to navigate apps, interpret app intent, and identify all relevant user flows. We are recording and storing the interaction chain of a test case in an intermediate representation.
+
+## Playwright generates test code
+
+After we record and store each test case's interaction chain, we generate the corresponding [Playwright code](https://playwright.dev/) deterministically on the fly immediately prior to test execution.
+
+Playwright code is exercising the UI. The interaction chain can be examined in the test detail view and the Playwright trace viewer.
```
Comment (sker65) [inline more-info/under-the-hood.mdx:18]: "Playwright code is exercising the UI" ?!? dont understand

File: changelog.mdx (hunk 1 above)
Comment (ma-zah) [reply changelog.mdx:55]: yeah, those were two different states. I covered it up in figma. will do in the next commit.

File: changelog.mdx (hunk 2 above)
Comment (ma-zah) [reply changelog.mdx:51]: took it word by word from the release, sounded like we know what we were doing. I'll change it.

Comment (ma-zah) [inline changelog.mdx:51]:
```suggestion
- `Open status for new test cases`: In order to mark test cases coming out of **suggest more** as new (until you've opened them for the first time), we introduced a new column **open status** .
```

File: changelog.mdx (hunk 4 above)
Comment (ma-zah) [reply changelog.mdx:127]: whaha... oops

File: changelog.mdx [hunk 7]
```diff
@@ -4,6 +4,144 @@ description: "All the nice stuff you asked for"
...
```
Comment (ma-zah) [inline changelog.mdx:127]:
```suggestion
- `Agent benchmark automation`: Every major change in the codebase impacts the AI agent performance. We get an automated ping whether the agent does what it's supposed to do, now.
```

File: more-info/octo-dictionary.mdx (hunk 5 above)
Comment (ma-zah) [reply more-info/octo-dictionary.mdx:84]: ah . brainfart

Comment (ma-zah) [inline more-info/octo-dictionary.mdx:84]:
```suggestion
### Visual locator picker
```

File: more-info/under-the-hood.mdx (hunk 6 above)
Comment (ma-zah) [reply more-info/under-the-hood.mdx:18]: That's a roedler formulation.  suggest a different formulation,.

Comment (sker65) [reply more-info/under-the-hood.mdx:18]: Sorry no idea, maybe we should himself ...

**Review by sker65 — APPROVED** [review]

File: more-info/under-the-hood.mdx (hunk 6 above)
Comment (ma-zah) [inline more-info/under-the-hood.mdx:18]:
```suggestion
The interaction chain can be examined in the test detail view and the Playwright trace viewer.
```

**Pull request merged**